### PR TITLE
fix(test): lint lll fix

### PR DESCRIPTION
**Pull request by skrdgraph**
## Problem
 `golangci-lint` tests are failing. The `lll` linter which addresses our `line-length` is reporting issues on a few files. 

## Solution
- fix up lll issues using `golines -m 120 -w <GO_FILE>`
- `golangci-lint run | grep lll` <- this will show where the tests are failing

#### Results 

`lint` error count reduces by ~50+ with this change. This only fixes `lll` issues

- ##### before
  ```
   > golangci-lint run | wc -l
  WARN [runner] The linter 'golint' is deprecated (since v1.41.0) due to: The repository of the linter has been archived by the owner.  Replaced by revive. 
       291
  ```
- ##### after
  ```
   > golangci-lint run | wc -l
  WARN [runner] The linter 'golint' is deprecated (since v1.41.0) due to: The repository of the linter has been archived by the owner.  Replaced by revive. 
       239
  ```

### CODE REVIEW

**Comment by skrdgraph** [comment]
@matthewmcneely @MichelDiz - I closed https://github.com/dgraph-io/dgraph/pull/8184, and reopened this one. This needed less changes because the previous `ignore` https://github.com/dgraph-io/dgraph/pull/8187 PR reduced the `lll` errors.

**Comment by skrdgraph** [comment]
converting to draft - as there are some issues on git commit history

File: graphql/resolve/query_rewriter.go [hunk 1]
```diff
@@ -1211,7 +1212,9 @@ func buildAggregateFields(
 				}
 				aggregateChild := &gql.GraphQuery{
 					Alias: aggregateField.DgraphAlias() + "_" + f.DgraphAlias(),
-					Attr:  strings.ToLower(function) + "(val(" + "" + f.DgraphAlias() + "_" + constructedForField + "Var))",
+					Attr: strings.ToLower(
+						function,
```
Comment (matthewmcneely) [inline graphql/resolve/query_rewriter.go:1216]: That's an odd reformat, I think I'd prefer:

```golang
				aggregateChild := &gql.GraphQuery{
					Alias: aggregateField.DgraphAlias() + "_" + f.DgraphAlias(),
					Attr: strings.ToLower(function) + 
                                             "(val(" + "" + f.DgraphAlias() + "_" + constructedForField + "Var))",
				}

```

File: graphql/schema/rules.go [hunk 2]
```diff
@@ -1951,10 +1951,19 @@ func customDirectiveValidation(sch *ast.Schema,
 			for _, h := range forwardHeaders.Children {
 				key := strings.Split(h.Value.Raw, ":")
 				if len(key) > 2 {
-					return append(errs, gqlerror.ErrorPosf(errPos,
+					return append(errs, gqlerror.ErrorPosf(
+						errPos,
+<<<<<<< HEAD
```
Comment (matthewmcneely) [inline graphql/schema/rules.go:1956]: Seems to be some unresolved conflicts here

File: worker/backup_ee.go [hunk 3]
```diff
@@ -1,3 +1,4 @@
+//go:build !oss
```
Comment (matthewmcneely) [inline worker/backup_ee.go:1]: I'm not familiar with this'go:build' type tag. Intentional?

File: graphql/schema/rules.go (hunk 2 above)
Comment (skrdgraph) [reply graphql/schema/rules.go:1956]: I messed up big time on this one trying to rebase something. I am fixing this up - give me a few mins

File: graphql/resolve/query_rewriter.go (hunk 1 above)
Comment (skrdgraph) [reply graphql/resolve/query_rewriter.go:1216]: sorry for the re-ask .. could you check this one https://github.com/dgraph-io/dgraph/pull/8191 (I have closed #8189 & opened #8190)

File: graphql/schema/rules.go (hunk 2 above)
Comment (skrdgraph) [reply graphql/schema/rules.go:1956]: sorry for the re-ask .. could you check this one https://github.com/dgraph-io/dgraph/pull/8191 (I have closed #8189 & opened #8190)

File: worker/backup_ee.go (hunk 3 above)
Comment (skrdgraph) [reply worker/backup_ee.go:1]: sorry for the re-ask .. could you check this one https://github.com/dgraph-io/dgraph/pull/8191 (I have closed #8189 & opened #8190)

**Pull request closed without merging**